components/omega/doc/design/OmegaV1GoverningEqns.md

```diff
@@ -0,0 +1,741 @@
+- **Omega-2.1: E3SM fully coupled** Ability to run G cases (active ocean and sea ice) and B cases (all components active) within E3SM. This will include: a full vertical mixing scheme, such as KPP; frazil ice formation in the ocean; and a submosescale parameterization. Omega 2.1 will mostly be run at eddy-resolving resolutions, and will not include parameterizations for lower resolutions such as Gent-McWilliams and Redi mixing.  Simulations will be compared to previous E3SM simulations, including [Caldwell et al. 2019](https://agupubs.onlinelibrary.wiley.com/doi/10.1029/2019MS001870) and [Petersen et al. 2019](https://agupubs.onlinelibrary.wiley.com/doi/abs/10.1029/2018MS001373).
+
+This document describes the governing equations for the layered ocean model, which are applicable for Omega-1.0 onwards. Specific terms, such as the pressure gradient, vertical mixing, and parameterizations, are left in their general form here but are described in more detail in other design documents.
+
+## 2. Requirements
+
+The requirements in the [Omega-0 design document](OmegaV0ShallowWater) still apply. Additional design requirements for Omega-1 are:
+
+### Omega will be an incompressible, hydrostatic, non-Boussinesq ocean model.
+See discussion in introduction. The non-Boussinesq formulation uses the full density throughout resulting in governing equations that conserve mass rather than volume. The substantial change is that the ``thickness'' variable, $h$, is now a mass-thickness, $h = \rho \Delta z$. This is explained in the derivation of the [layered equations below](#layered-equations).
+
+### Omega will use TEOS10 for the equation of state.
+See additional [EOS design document](EOS)
+
+### Omega-1.0 will add new terms for the pressure gradient, vertical mixing, and vertical advection.
+See forthcoming design documents on the pressure gradient, vertical mixing, and vertical advection.
+
+## 3. Continuous Equations
+
+The continuous form of the conservation equations are as follows [Kundu et al. 2024](https://www.amazon.com/Fluid-Mechanics-Pijush-K-Kundu/dp/012405935X), chapter 4, eqns 4.7 and 4.22. This is before any assumptions are made, so this is a compressible, non-hydrostatic, non-Boussinesq fluid. Here all variables are a function of $(x,y,z)$, ${\bf u}_{3D}$ denotes the three-dimensional velocity vector, ${\bf u}_{3D} \otimes {\bf u}_{3D} = {\bf u}_{3D}{\bf u}_{3D}^T$ is the tensor product, $\nabla_{3D}$ is the three-dimensional gradient, $D/Dt$ is the material derivative, and other variables defined in the [Variable Definition Section](#variable-definitions) below.
+
+momentum:
+
+$$
+\frac{D \rho {\bf u}_{3D} }{D t} \equiv
+\frac{\partial \rho {\bf u}_{3D}}{\partial t}
+ + \nabla_{3D} \cdot \left( \rho {\bf u}_{3D} \otimes {\bf u}_{3D}  \right)
+  = - \nabla_{3D} p
++ \rho {\bf D}^u + \rho {\bf F}^u
+$$ (continuous-momentum)
```

Comment:
the buoyancy term is missing here `rho g khat`

Reply:
The first equation set is the most general, so gravity is within ${\bf F}^u$. A few paragraphs down I begin "Geophysical fluids such as the ocean and atmosphere are rotating and stratified..." and start to make assumptions and call out specific terms. I have the sentence "The gravitational force $g {\bf k}$ is included in ${\bf F}^u$".

Reply:
oh sorry, totally missed that.  Thanks for the explanation